cmd/opampsupervisor/supervisor/templates/owntelemetry.yaml

```diff
@@ -1,21 +1,10 @@
-receivers:
-  # Collect own metrics
-  prometheus/own_metrics:
-    config:
-      scrape_configs:
-        - job_name: 'otel-collector'
-          scrape_interval: 10s
-          static_configs:
-            - targets: ['0.0.0.0:{{.PrometheusPort}}']  
-exporters:
-  otlphttp/own_metrics:
-    metrics_endpoint: "{{.MetricsEndpoint}}"
 
 service:
   telemetry:
     metrics:
-      address: ":{{.PrometheusPort}}"
-  pipelines:
-    metrics/own_metrics:
-      receivers: [prometheus/own_metrics]
-      exporters: [otlphttp/own_metrics]
+      readers:
```

Comment:
Is there any test that confirms this functionality? I'm looking for a test that would confirm metric names, endpoint, and format are all as expected.

Reply:
Might be worth opening a follow up issue to ensure metrics are emitted with report_own_metrics enabled

Reply:
We really should have a test to confirm this in the future, but for now this should at least be a no-op change from the users' perspective outside no longer needing a Prometheus receiver to use the Supervisor to get the Collector's own metrics.

I think it's okay to rely on the fact that the current test suite passes to rely on this not breaking anything.